homeassistant/components/control4/media_player.py

```diff
@@ -0,0 +1,406 @@
+"""Platform for Control4 Rooms."""
```

Comment:
Can we come up with a more descriptive name than just "room"? Same comment for the media player sub class below.

Reply:
In control4 they are called rooms. The media control centers around room based audio so rooms is the correct description